### ISSUE TITLE
Problem setting up quick-add shortcut

### ISSUE THREAD

**Issue by YanniPapandreou** [alainm23/planify#886]
**Describe the bug**

Thanks for all the great work with Planner!

I used to be able to create a custom keyboard shortcut to launch quick-add in planner when I was using Ubuntu 20.04. I have since changed my OS to Pop!_OS 21.10 and now this no longer works. I have tried running all the following commands from my terminal and they all do not work and give the following errors:

```fish
~ ➜ flatpak run com.github.alainm23.planner-quick-add
error: app/com.github.alainm23.planner-quick-add/x86_64/master not installed
~ ✖ flatpak run --command=com.github.alainm23.planner-quick-add --own-name=com.github.alainm23.planner-quick-add com.github.alainm23.planner
bwrap: execvp com.github.alainm23.planner-quick-add: No such file or directory
~ ✖ flatpak run --talk-name=org.gnome.settings-daemon.plugins.media-keys com.github.alainm23.planner.quick.add
error: app/com.github.alainm23.planner.quick.add/x86_64/master not installed
~ ✖ flatpak run --talk-name=org.gnome.settings-daemon.plugins.media-keys com.github.alainm23.planner-quick-add
error: app/com.github.alainm23.planner-quick-add/x86_64/master not installed
~ ✖ flatpak run --command=com.github.alainm23.planner.quick-add com.github.alainm23.planner
bwrap: execvp com.github.alainm23.planner.quick-add: No such file or directory
```

**Expected behavior**

On my old OS the terminal command `flatpak run --command=com.github.alainm23.planner.quick-add com.github.alainm23.planner` used to work and launched quick-add. This no longer works. Is there a new command I need to run?

**Desktop:**
 - OS or DE: Pop!_OS 21.10 x86_64 with Gnome 40.5
 - Version of Planner: 3.0.6 (flathub/flatpak)

**Comment by davelbit** [comment]
Any updates on this matter? I was also wondering about that.

**Comment by alainm23** [comment]
Planner is being ported to [Gtk4 and libadwaita](https://github.com/alainm23/planner/tree/planner-gnome). Quick Add is in the development plans. Stay tuned, a beta will be announced very soon.

**Comment by alainm23** [comment]
Hello, Quick Add is available in Planify 4.1, a keyboard shortcut preset is required (Until Global Keyboard portal is available) by the end user. Greetings.

https://flathub.org/es/apps/io.github.alainm23.planify
